public/chore.js

```diff
@@ -29,17 +29,17 @@ choreList.forEach(function (chore) {
     list.appendChild(li);
 });
 
-const app = document.querySelector('#app');
-app.appendChild(list);
-
 
-function addChore() {
+function addChore() { 
+    const newLi = document.createElement('li');
     const newChore = {
         name: document.getElementById('chore').value,
         assignee: document.getElementById('assigned').value
     }
-        choreList.push(newChore);
-        console.log(choreList);
+    newLi.textContent = `Chore: ${newChore.name} - Assigned to: ${newChore.assignee}`;
+    // choreList.push(newChore);
+    list.appendChild(newLi);
     return false;
-
 }
+const app = document.querySelector('#app');
```

Comment:
I think you should move this back up to where it was. Because it is outside of a function, it will run when the page loads so we don't want to hide it at the bottom of the file.